### DISCUSSION TITLE
Welcome to TaraniTunes-v4.x Discussions!

### DISCUSSION THREAD

**Discussion by jrwieland** [jrwieland/TaraniTunes-v4.x#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
    * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with your comments or questions.

**Comment by 2tensy1** [comment]
Can i execute this one on my radiomaster pocket?

**Reply by jrwieland** [reply]
Yes, the pocket I believe is a monochrome screen, so use the Black and white version
if you have any issues email me direct john.r.wieland@gmail.com
